### DISCUSSION TITLE
Passing physical device to virtualisation.

### DISCUSSION THREAD

**Discussion by JayDoubleu** [okd-project/okd#1011] [Q&A]
Is there a way to pass physical devices to a VM ?

**Comment by rvanderp3** [comment]
@JayDoubleu What hypervisor are you using?  I would expect there to be a facility that the hypervisor provides to enable that.  From the perspective of OKD, the [docs](https://docs.okd.io/latest/hardware_enablement/about-hardware-enablement.html) describe how one might use that hardware in a cluster.

**Reply by JayDoubleu** [reply]
Apologies, I should be more precise. My installation is already on bare metal (single node)
What I would like to do, is attach physical disk device from the host into virtual machine created via OpenShift Virtualization operator.

**Reply by rvanderp3** [reply]
No worries at all.  I believe https://docs.okd.io/latest/virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.html may be what you are trying to achieve.

**Reply by JayDoubleu** [reply]
I see. Do I understand correctly that hostpath is a mountpoint under which will be used for storage ?
I'm actually trying to do a "passthrough" similar to podman --device so I can see that entire block device lets say /dev/sdb inside the virtual machine.
